recipes/r-ldweaver/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "r-ldweaver" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "4369b95bafd128a63f10b2288fbce66deb685e22d0b7be307baaefe5005a3356" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Sudaraka88/LDWeaver/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - {{ pin_subpackage('r-ldweaver', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('cxx') }}            # [not win]
+
+  host:
+    - r-base 4.2
+    - bioconductor-genomicranges
+    - r-matrix
+    - r-matrixextra
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-ape
+    - r-chromomap
+    - r-data.table
+    - r-dplyr
+    - r-fitdistrplus
+    - bioconductor-genbankr
+    - r-ggnewscale
+    - r-ggplot2
+    - r-ggraph
+    - bioconductor-ggtree
+    - r-heatmap3
+    - r-htmlwidgets
+    - r-igraph
+    - r-phytools
+    - r-plyr
+    - r-stringi 1.7.12
```

Comment:
Same here do we need this strict version or can we relax this pin a bit?

Reply:
Relaxed